web/themes/custom/sfgovpl/src/sass/node/_node-news.scss

```
@@ -3,28 +3,28 @@
   @include rubik;
   color: $c-slate;
 
-  .news--meta,
-  .news--image,
-  .news--body {
-    &-inner {
-      margin: 0 auto;
-      max-width: $w-news-body-container;
-      padding-left: 20px;
-      padding-right: 20px;
-      position: relative;
-
-      @include media($narrow-screen) {
-        padding-left: 96px;
-        padding-right: 96px;
-      }
-      > * {
-        @include media($narrow-screen) {
-          padding-right: 40px;
-          width: ((strip-unit($w-news-body) + 40) / strip-unit($w-news-body-content)) * 100%;
-        }
-      }
-    }
-  }
+  // .news--meta,
+  // .news--image,
+  // .news--body {
+  //   &-inner {
+  //     margin: 0 auto;
+  //     max-width: $w-news-body-container;
+  //     padding-left: 20px;
+  //     padding-right: 20px;
+  //     position: relative;
+
+  //     @include media($narrow-screen) {
+  //       padding-left: 96px;
+  //       padding-right: 96px;
+  //     }
+  //     > * {
+  //       @include media($narrow-screen) {
+  //         padding-right: 40px;
+  //         width: ((strip-unit($w-news-body) + 40) / strip-unit($w-news-body-content)) * 100%;
+  //       }
+  //     }
+  //   }
+  // }
```

can we get rid of this instead of commenting out? hooray for removing css!
```suggestion
  // .news--meta,
  // .news--image,
  // .news--body {
  //   &-inner {
  //     margin: 0 auto;
  //     max-width: $w-news-body-container;
  //     padding-left: 20px;
  //     padding-right: 20px;
  //     position: relative;

  //     @include media($narrow-screen) {
  //       padding-left: 96px;
  //       padding-right: 96px;
  //     }
  //     > * {
  //       @include media($narrow-screen) {
  //         padding-right: 40px;
  //         width: ((strip-unit($w-news-body) + 40) / strip-unit($w-news-body-content)) * 100%;
  //       }
  //     }
  //   }
  // }
```